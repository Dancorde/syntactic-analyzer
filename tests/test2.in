program lalg;
{outro teste}
var a, minha_variavel: integer;
begin
readd(a, @, 1.a23);
end.