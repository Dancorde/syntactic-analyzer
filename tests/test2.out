program - program
lalg - id
; - ;
var - var
a - id
, - ,
minha_variavel - erro - identificador muito grande
: - :
integer - integer
; - ;
begin - begin
readd - id
( - (
a - id
, - ,
@ - erro - simbolo n√£o pertence a linguagem
, - ,
1.a23 - erro - numero real mal formatado
) - )
; - ;
end - end
. - .
